app/views/govuk_publishing_components/components/_radio.html.erb

```diff
@@ -40,9 +42,9 @@
 
     <% if heading.present? %>
       <% if is_page_heading %>
-        <%= tag.legend class: "govuk-fieldset__legend govuk-fieldset__legend--xl gem-c-title" do %>
-          <%= tag.span(heading_caption, class: "govuk-caption-xl") if heading_caption.present? %>
-          <%= tag.h1 heading, class: "gem-c-title__text" %>
+        <%= tag.legend class: legend_classes do %>
+          <%= tag.span(heading_caption, class: "govuk-caption-#{heading_size}") if heading_caption.present? %>
+          <%= tag.h1 heading, class: "gem-c-title__text govuk-heading-#{heading_size}" %>
```

Comment:
To make a legend large, you should use `govuk-fieldset__legend--l` on the legend, and the heading should just have `govuk-fieldset__heading`.

See https://design-system.service.gov.uk/get-started/labels-legends-headings/ for info.

Reply:
Done